roles/yoda_rulesets/templates/job_movetovault.r.j2

```diff
@@ -13,22 +13,41 @@
 #
 
 uuYcRunIntake2Vault {
-	# intake areas can be added to the grouplist as needed
-	*grouplist = list ( {{ intake_groups | map("regex_replace","(.+)",'\"\\1\"') | join(',') }} );
-	*zone = $rodsZoneClient;
+        # intake areas can be added to the grouplist as needed
+        *grouplist = list ( {{ intake_groups | map("regex_replace","(.+)",'\"\\1\"') | join(',') }} );
+        *grp_prefixes = list ("grp-intake-", "intake-");
+        *zone = $rodsZoneClient;
 
-	foreach (*grp in *grouplist) {
-		*intakeRoot = "/*zone/home/grp-intake-*grp";
-		*vaultRoot  = "/*zone/home/grp-vault-*grp";
-		uuLock(*vaultRoot, *status);
-		if (*status == 0) {
-			# we have a lock
-			uuYc2Vault(*intakeRoot, *vaultRoot, *status);
-			if (*status == 0 ) then *result = "ok" else *result = "ERROR (*status)";
-			writeLine("serverLog","RunIntake2Vault for *intakeRoot result = *result");
-			uuUnlock(*vaultRoot);
-		}
-	}
+        foreach (*grp in *grouplist) {
+                *intakeRoot = "";
+                *found = false;
+
+                foreach (*prefix in *grp_prefixes) {
+                    *grp_name = *prefix ++ *grp;
+                    foreach(*row in SELECT USER_GROUP_NAME
+                        WHERE USER_GROUP_NAME = '*grp_name'
+                        AND USER_TYPE = 'rodsgroup') {
+
+                        *path=*row.USER_GROUP_NAME;
+                        *found = true;
+                        break;
+                    }
+                    if (*found) {
+                        break;
+                    }
+                }
+                *intakeRoot = "/*zone/home/*grp_name";
+                *vaultRoot  = "/*zone/home/grp-vault-*grp";
```

Comment:
This results in a rule failure on groups with an "intake-" prefix, since groups with this prefix use "vault-" rather than "grp-vault" as their prefix.

This probably also applies to the other copy of this script in the yoda_test role